purchase_sale_stock_inter_company/models/stock_move_line.py

```diff
@@ -0,0 +1,95 @@
+from odoo import api, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        new_move_lines = super().create(vals_list)
+        for move_line in new_move_lines:
+            purchase_line_origin = (
```

Comment:
Is the direction OK from sales to purchase instead of the reverse?

Reply:
But the use case is not important here. Having MTO doesn't matter. The question is that you should receipt the purchase, and that synchronize the sales, not the contrary.

Reply:
In this case, the user must fill in the lot in the sale order of the company where the delivery is made. If I sync from the purchase to the sale, how can I determine which product lot it is? Sorry, but I don’t understand your point.

Reply:
Yes, my fault, you are right. Delivery picking triggers the synchronization to receipt picking.